vizro-core/docs/pages/user_guides/custom_charts.md

```diff
@@ -65,13 +70,19 @@ Building on the above, there are several routes one can take. The following exam
                 ),
             ],
             controls=[
-                vm.Filter(column="petal_width"),
+                vm.Parameter( # (2)!
```

Comment:
This is a really nice example. Should we put in an animated gif just to make it completely obvious what's possible with this example?

Or in future maybe we would have this in our "feature showcase" dashboard and just link to that.

Reply:
Good point. I added a section, and also a GIF. I do think that this actually belongs also to a feature showcase, as we do not use GIFs anywhere else, but this will do for now